workflows/toast_sim_ground.py

```diff
@@ -0,0 +1,311 @@
+    # by estimating the data volume.
+
+    group_size = toast.job_group_size(
+        world_comm,
+        jobargs,
+        schedule=schedule,
+        focalplane=focalplane,
+        full_pointing=full_pointing,
+    )
+
+    # Create the toast communicator
+
+    comm = toast.Comm(world=world_comm, groupsize=group_size)
+
+    # Create the (initially empty) data
+
+    data = toast.Data(comm=comm)
+
+    # Create a telescope for the simulation.  Again, for a specific experiment we
+    # would use custom classes for the site.
+
+    site = toast.instrument.GroundSite(
+        schedule.site_name, schedule.site_lat, schedule.site_lon, schedule.site_alt
+    )
+    telescope = toast.instrument.Telescope(
+        schedule.telescope_name, focalplane=focalplane, site=site
+    )
+
+    # Simulate the telescope pointing
+
+    if not ops.sim_ground.enabled:
+        msg = "Cannot disable the satellite scanning operator"
+        if rank == 0:
+            log.error(msg)
+        raise RuntimeError(msg)
+
+    ops.sim_ground.telescope = telescope
+    ops.sim_ground.schedule = schedule
+    ops.sim_ground.apply(data)
+
+    # Construct a "perfect" noise model just from the focalplane parameters
+
+    ops.default_model.apply(data)
+
+    # Create the Elevation-weighted noise model
+
+    ops.elevation_model.detector_pointing = ops.det_pointing
+    ops.elevation_model.view = ops.det_pointing.view
+    ops.elevation_model.apply(data)
+
+    # Set the pointing matrix operators to use the detector pointing
+
+    ops.pointing.detector_pointing = ops.det_pointing
+    ops.pointing_final.detector_pointing = ops.det_pointing
+
+    # Simulate sky signal from a map.  We scan the sky with the "final" pointing model
+    # if that is different from the solver pointing model.
+
+    if ops.scan_map.enabled:
+        pix_dist = toast.ops.BuildPixelDistribution()
+        if ops.binner_final.enabled and ops.pointing_final.enabled:
+            pix_dist.pixel_dist = ops.binner_final.pixel_dist
+            pix_dist.pointing = ops.pointing_final
+            pix_dist.shared_flags = ops.binner_final.shared_flags
+            pix_dist.shared_flag_mask = ops.binner_final.shared_flag_mask
+            pix_dist.save_pointing = ops.binner_final.full_pointing
+        else:
+            pix_dist.pixel_dist = ops.binner.pixel_dist
+            pix_dist.pointing = ops.pointing
+            pix_dist.shared_flags = ops.binner.shared_flags
+            pix_dist.shared_flag_mask = ops.binner.shared_flag_mask
+            pix_dist.save_pointing = full_pointing
+        pix_dist.apply(data)
+
+        ops.scan_map.pixel_dist = pix_dist.pixel_dist
+        ops.scan_map.pointing = pix_dist.pointing
+        ops.scan_map.apply(data)
+
+    # Simulate detector noise
+
+    if ops.sim_noise.enabled:
+        ops.sim_noise.apply(data)
+
+    # Build up our map-making operation from the pieces- both operators configured
+    # from user options and other operators.
+
```

Comment:
This workflow is still missing all support for filter+bin. That is fine for now but it would be helpful to add comments to where that would go in the file.